### PR TITLE
[wip] Fix pulled application status

### LINKED ISSUE #270: fix pull application status on Recipe page when environment managed from Environments page

### PR DESCRIPTION
**Pull request by feloy**
### What does this PR do?

This PR merges the Environment manager into the Application manager. 

### What issues does this PR fix or reference?

Fixes #270

### How to test this PR?

Play with environments (remove, restart) or pods (remove) and check if the status is corrected reflected in the recipe page.

### CODE REVIEW

File: packages/backend/src/studio-api-impl.ts [hunk 1]
```diff
@@ -181,6 +181,32 @@ export class StudioApiImpl implements StudioAPI {
       });
   }
 
+  async requestRestartEnvironment(recipeId: string): Promise<void> {
```
Comment (axel7083) [inline packages/backend/src/studio-api-impl.ts:184]: is it rebase ?

**Comment by axel7083** [comment]
I tested it locally, it is working as expected

I tried 
- Stopping from the pod list ✅
- Stopping from CLI ✅

What is still _weird_ to me
- We have the `running` status in the environment page but still starting in recipe page

**Comment by feloy** [comment]
> 
> What is still _weird_ to me
> 
> * We have the `running` status in the environment page but still starting in recipe page

Yes, I agree. This is because the env page does not (want to) use the state from the application manager (which is able to know the 'loading' status), which will disappear with #265. It should be fixed with #265

**Comment by lstocchi** [comment]
I'm always a bit confused when i read WIP in the title as i don't get if it's ready to be tested or not. Could we use the draft option in gh to mark a PR as wip?
@feloy Is this ready to be reviewed?

**Comment by feloy** [comment]
> I'm always a bit confused when i read WIP in the title as i don't get if it's ready to be tested or not. Could we use the draft option in gh to mark a PR as wip? @feloy Is this ready to be reviewed?

No, this one is not for review. I'll most probably replace it by another one with a different refactoring, based on what we discussed this morning

**Comment by jeffmaury** [comment]
> > I'm always a bit confused when i read WIP in the title as i don't get if it's ready to be tested or not. Could we use the draft option in gh to mark a PR as wip? @feloy Is this ready to be reviewed?
> 
> No, this one is not for review. I'll most probably replace it by another one with a different refactoring, based on what we discussed this morning

Please use draft status in that case, I have the same concern

**Comment by feloy** [comment]
closed in favor of #289

**Pull request closed without merging**